vulnfeeds/cmd/cve-bulk-converter/main.go

```diff
@@ -0,0 +1,114 @@
+// Package main converts Linux CVEs to OSV format in bulk.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/google/osv/vulnfeeds/cmd/cvelist2osv"
+	"github.com/google/osv/vulnfeeds/cves"
+	"github.com/google/osv/vulnfeeds/utility/logger"
+)
+
+type PartialCVE struct {
+	Metadata struct {
+		ID       string `json:"cveId"`
+		Assigner string `json:"assignerShortName"`
+		State    string `json:"state"`
+	} `json:"cveMetadata"`
+}
+
+func main() {
+	// --- Configuration ---
+	years := flag.String("years", "2022,2023,2024,2025", "A comma-separated list of years to process.")
+	workers := flag.Int("workers", 10, "The number of concurrent workers to use for processing CVEs.")
```

Comment:
You might be able to bump this up by quite a bit, depending on how much time is spent reading from disk.